qemu/tests/cfg/numa_opts.cfg

```diff
@@ -18,84 +20,47 @@
           numa_nodes = 1
```

Comment:
Could you explain why this node do not need `guest_numa_nodes` ?

Reply:
This case means to not set numa node in cli, so we don't need guest_numa_nodes.

Reply:
Ok, thanks for your explanation.